src/test/java/net/snowflake/client/jdbc/SnowflakeDriverIT.java

```diff
@@ -2655,10 +2656,17 @@ public void testClientInfo() throws Throwable {
 
       JsonNode clientInfoJSON = mapper.readTree(clientInfoJSONStr);
 
-      // assert that spart version and spark app are found
-      assertEquals("spark version mismatch", "1.2.0", clientInfoJSON.get("sparkVersion").asText());
+      // assert that spark version and spark app are found
```

Comment:
@sfc-gh-mrui , this is where we test that Test System.setPropertye() also works like a connection property. I updated it to match with the JSON string you gave me.